frontend/src/libraries/language.ts

```diff
@@ -81,205 +30,105 @@ export const parseAcceptLanguage = (input: string): string[] => {
   return input.split(",").map((tag) => tag.split(";")[0]);
 };
 
-export function getBrowserLocale(appLanguages: string[]): string | undefined {
-  const browserLanguages = getUserLocales();
-
-  debugLog("Languages from browser:", browserLanguages);
-
-  if (
-    appLanguages &&
-    appLanguages.length > 0 &&
-    browserLanguages &&
-    browserLanguages.length > 0
-  ) {
-    return findBestSupportedLocale(appLanguages, browserLanguages);
-  }
-}
-
-export function getAcceptedLocale(
-  appLanguages: string[],
+export function getAcceptedLanguage(
```

Comment:
We probably should move to something more error-prone, like language detectors provided by `i18next` (in [browser](https://github.com/i18next/i18next-browser-languageDetector), or in [server](https://github.com/i18next/i18next-http-middleware#language-detection) environment)